src/weaponstats.cc

```diff
@@ -0,0 +1,171 @@
+#include "weaponstats.hh"
+#include "functions.hh"
+#include <cstdlib>
+#include <algorithm>
+#include "injector/injector.hpp"
+#include "logger.hh"
+#include "base.hh"
+#include <iostream>
+#include "config.hh"
+
+int                    statsStart                       = 0xC8AAB8;
```

Comment:
This is better suited to be in `functions.hh`. You can also replace this with a struct pointer, I'll describe that in detail below.

Basically, you put this in functions.hh (See below for CWeaponInfo definition)
```cpp
static CWeaponInfo* aWeaponInfos;
```
and this at the end of functions.cc
```cpp
CWeaponInfo* aWeaponInfos = 0xC8AAB8
```